client/state/posts/selectors.js

```diff
@@ -359,6 +363,26 @@ export const isEditedPostDirty = createSelector(
 );
 
 /**
+  * Returns true if the edited post has content
+  * (title, excerpt or content not empty)
+  *
+  * @param  {Object}  state  Global state tree
+  * @param  {Number}  siteId Site ID
+  * @param  {Number}  postId Post ID
+  * @return {Boolean}        Whether the edited post has content or not
+ */
+export function editedPostHasContent( state, siteId, postId ) {
+	const editedPost = getEditedPost( state, siteId, postId );
+	return (
+		!! editedPost &&
+		(
+			some( [ 'title', 'excerpt' ], ( field ) => editedPost[ field ] && !! editedPost[ field ].trim() ) ||
+			! isEmptyContent( editedPost.content )
```

Comment:
Eventually we'll want to put some thought into how we're going to pass and test against a post's raw content, from the existing `PostEditStore` (where raw content is updated more frequently for dirty checking but is not the value we send to the API).